app/controllers/support_interface/sessions_controller.rb

```diff
@@ -3,6 +3,8 @@ class SessionsController < SupportInterfaceController
     skip_before_action :authenticate_support_user!, except: :destroy
 
     def new
+      redirect_to support_interface_path if current_support_user
```

Comment:
Do we need to add an early return here?

```suggestion
      redirect_to support_interface_path and return if current_support_user
```